### ISSUE TITLE
Bug when logging in

### ISSUE THREAD

**Issue by AlgyTaylor** [fguillot/kanboard#780]
If you use IE7* to log in, regardless of what details you enter it returns you to the login page with "The username is required" and "The password is required" error messages.

I've also been told that the same problem exists on an iPad using either Chrome or Safari, so it's not just an MS-thing .. but certainly I can recreate the problem on MSIE7.

**Comment by s-boardman** [comment]
I've had this issue on iPad running iOS 8.2 using Chrome v41.0.2272.58 and Safari.

**Comment by fguillot** [comment]
What is the os/php/version on the server? because I use Kanboard with my iPad without any problem.

**Comment by AlgyTaylor** [comment]
Thanks for the reply ... my stack is Windows Server 2008 / Apache / PHP 5.5.15 / PostgreSQL :)

**Comment by mfoucrier** [comment]
I'm using Kanboard 1.012 with Ubuntu server 14.04 / Apache / PHP 5.x / PostgreSQL 9.x.
And I has the same problem, if I connect later after launching Firefox. I have to re-enter the user and password in this second login page. But, if I connect immediately after launching Firefox, the connection is correct with the first login page!

**Comment by fguillot** [comment]
I have to investigate and setup a vm to find the issue.

**Comment by fguillot** [comment]
@AlgyTaylor I fixed a couple of bugs about Windows installations, the authentication was broken if you installed kanboard without a subfolder and if you use Internet Explorer. This issue is now fixed in the development version, the fix is here if you don't want to wait the next stable version: https://github.com/fguillot/kanboard/commit/81841c73c2fd05f3274fa790141aa9b838db9dd3

I will do more tests on Windows platforms over the next days (Kanboard is mainly used on Linux/Unix servers).

Note: I'm testing only with IE11, not older versions.
